src/templates/Home/PostPreview.module.scss

```diff
@@ -0,0 +1,117 @@
+@use '../../styles/variables';
+
+.container {
+  display: grid;
+  grid-template-areas: 'content';
+}
+
+.containerFeatured {
+  display: grid;
+  gap: var(--mm-spacing);
+  grid-template-areas: 'graphic'  'content';
```

Comment:
Should be `'graphic content'`

Reply:
For mobile, I intended for this to be stacked, not side by side.

Reply:
I added the linebreak.